config/assembly-bin.xml

```diff
@@ -19,10 +19,6 @@
     <fileSet>
       <useDefaultExcludes>true</useDefaultExcludes>
       <includes>
-        <include>config/checkstyle_checks.xml</include>
-        <include>config/import-control.xml</include>
-        <include>config/suppressions.xml</include>
-        <include>config/java.header</include>
```

Comment:
Files are internal and not really need for binary distribution. This is more for running internal tests.